rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import { terser } from 'rollup-plugin-terser';
+
+
+// Production state is based on whether we are watching for changes or not.
+const production = !process.env.ROLLUP_WATCH;
+
+export default [
+    {
+        input: 'src/ts/main.ts',
+        output: {
+            file: 'dist/js/main.min.js',
+        },
+        plugins: [
```

Comment:
We lost the notifier; looks like this guy is a rollup equivalent. Can you check it out and if it works well, add it?

https://github.com/MikeKovarik/rollup-plugin-notify

Reply:
I've added it but it won't notify any sass errors.  This is because sass is compiled outside the rollup build. 

To get sass warnings, I think we need to build sass via gulp and add that.  Similar to this: https://gist.github.com/lmartins/cebf7d01992e18888ecf#file-gulpfile-js-L20

Do we want to add gulp or just keep it simple as is?

Reply:
Ah got it; no let's keep it as-is. We can investigate better error support for Sass in the future. I think not using Gulp is a good idea. Thanks!